meadow/src/Git.hs

```diff
@@ -1,4 +1,28 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE TemplateHaskell #-}
 module Git where
```

Comment:
This could be in its own package, right? I'm envisaging trying to extract this out into `iohk-nix` or something. That would look like depending on the magic package, then using the nix function to compile statically against it and replace the string.

Reply:
I'm not sure TBH, I'll have a go at extracting it into a new package in the plutus project to see if it works

Reply:
ok, I've extracted to the `interpreter` package. It's not exactly the right place but it is commonly shared by both playgrounds and I didn't really want to set up an entire `utils` package just for those 2 modules. Not sure what else to do. WRT `iohk-nix` the problem is that it doesn't currently have cabal haskell projects that we would want to work with stack, I'm not sure if it belongs in there really, not sure where it belongs TBH.

Reply:
I was thinking literally a `git-rev` package but maybe that's overkill.

Reply:
Also we possibly should have `utils`, there's a ton of stuff that's in `language-plutus-core` that would belong somewhere like that.

Reply:
But we can do that separately.

Reply:
Yeah, I think overkill, although it seems more correct. In one way it would make sense to have a public repo with this code and the nix code that it is inherently linked to however actual use of those 2 parts would not be practical so for such a small amount of code I see copy and paste as being the better solution.